canonical/canonical.com
# Redesign documentation landing and subpages

## Done

- Applied redesign as per [mock-up](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9999-32699&t=jovsgOJ31F3rH5s1-0)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-905.demos.haus/documentation
- Check each tab against design and see that it matches

## Issue / Card

Fixes
